### PR TITLE
Direct debit guides

### PR DESCRIPTION
**Pull request by ZackJaime**
Some guides for direct debits. Viewable [here](https://wealthkernel.stoplight.io/docs/guides/branches/Direct-Debit-Guides/docs/direct-debits/Api-Access.md)
However, you may need to request access to view unpublished branches.

### CODE REVIEW

**Comment by dantheman999301** [comment]
Comment related but not to a line, could we add something to the "Welcome page" as a table of contents to the different sections.

A bit like this - https://meta.stoplight.io/docs/platform/ZG9jOjIwNjk2MQ-welcome-to-the-stoplight-docs

File: docs/direct-debits/Api-Access.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+tags: [Direct Debits]
+---
+
+# Getting Access to the API
+
+All Direct Debit APIs are available in the sandbox environment. To access these APIs in Live you must complete the following steps to meet compliance requirements:
```
Comment (dantheman999301) [inline docs/direct-debits/Api-Access.md:7]:
```suggestion
All Direct Debit APIs are available in the sandbox environment. To access these APIs in Production you must complete the following steps to meet compliance requirements:
```

File: docs/direct-debits/Api-Access.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+1. You must agree to our commercial terms and legal arrangements covering the usage of the Direct Debit facility.
+2. You must have updated your retail customer Terms & Conditions to include the Direct Debit payment service.
+3. The UI of the customer facing screens used to set up a Direct Debit mandate must be approved based on the  [guidelines](./UI-Guidelines.md) outlined below.
```
Comment (dantheman999301) [inline docs/direct-debits/Api-Access.md:11]:
```suggestion
3. The UI of the customer facing screens used to set up a Direct Debit mandate must be approved based on the  [guidelines](./UI-Guidelines.md) outlined.
```

Assume the below is based on when these were all one flowing document.

File: docs/direct-debits/How-To-Request-Payments.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+The bank account number and sort code used to create the mandate must be able to pass a Modulus check. To test on sandbox  you should use `55779911` as the account number and `20-00-00` as the sort code.
+
+Once you've met the prerequisites you can request a new mandate using the <a href="/docs/api/docs/openapi/api.yaml/paths/~1direct-debits~1mandates/post">Create Mandate API</a>.
+
+When you create a mandate it starts with a `Pending` status. You cannot create payments until it has transitioned into an 'Active' status. It's also at this stage you would display the success page referred to in the [UI Guidelines](./Api-Access.md#page-3---success-screen). You can monitor this using the returned mandate Id against the <a href="/docs/api/docs/openapi/api.yaml/paths/~1direct-debits~1mandates~1%7BmandateId%7D/get">Get Mandate API</a>.
+
+The end user will receive an email, usually within an hour, of the mandate being create. This email have the mandate PDF attached to it.
+
+> It's still possible for a mandate to fail after it's reached the Active state. If this happens, any payments created against the mandate will failed as well.
```
Comment (dantheman999301) [inline docs/direct-debits/How-To-Request-Payments.md:18]: Just as a styling not, if you want different colours for these (say as a warning) then you can use this:

https://meta.stoplight.io/docs/studio/ZG9jOjg0-stoplight-flavored-markdown#block-types

Comment (ZackJaime) [reply docs/direct-debits/How-To-Request-Payments.md:18]: Thanks for the suggestion, I did see those, but I think simply highlighting them is enough.

**Review by dantheman999301 — APPROVED** [review]

**Pull request merged**